src/useSnapEdgeToParent/helpers.ts

```diff
@@ -13,12 +13,12 @@ export interface Refs<
 export type Vertical = 'top' | 'bottom'
 export type Horizontal = 'left' | 'right'
 
-export interface AutoLayout {
+export interface IAutoLayout {
```

Comment:
Can you add a ticket in jira to improve this name?

Reply:
later === never, open to suggestions now!

Reply:
Renamed to AutoPosition